src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Await.cs

```diff
@@ -16,7 +18,7 @@ public override BoundNode VisitAwaitExpression(BoundAwaitExpression node)
 
         public BoundExpression VisitAwaitExpression(BoundAwaitExpression node, bool used)
         {
-            return RewriteAwaitExpression((BoundExpression)base.VisitAwaitExpression(node), used);
+            return RewriteAwaitExpression((BoundExpression)base.VisitAwaitExpression(node)!, used);
```

Comment:
Can base be annotated better?

Reply:
No.  The base permits a return of null because some overrides do.  We could insert an intermediate class, but that would spam the callstack.

---
In reply to: [386597010](https://github.com/dotnet/roslyn/pull/42068#discussion_r386597010) [](ancestors = 386597010)